pmda/parallel.py

```diff
@@ -404,23 +401,23 @@ def run(self,
         self.timing = Timing(
             np.hstack([el[1] for el in res]),
             np.hstack([el[2] for el in res]), total.elapsed,
-            np.array([el[3] for el in res]), time_prepare,
+            time_prepare,
+            time_prepare_dask,
             conclude.elapsed,
             # waiting time = wait_end - wait_start
-            np.array([el[4]-wait_start for el in res]),
-            np.array([el[5] for el in res]),
-            np.array([el[6] for el in res]))
+            np.array([el[3] - wait_start for el in res]),
+            np.array([el[4] for el in res]),
+            np.array([el[5] for el in res]))
+
+        #  this is crucial if the analysis does not iterate over
```

Comment:
Because--no sure it should be defined as a bug--
e.g. Density Analysis (both in MDAnalysis and this PR) depends on the current ts of the universe. 
```python
def _prepare(self):
        coord = self._select_atomgroup.positions  #   It will change with ts.
        ...
```

Reply:
And currently, the universe will stay its ending frame after analysis if not being rewinded.

Reply:
oh because this does not return a copy. I would not do the rewind. If people want a copy they should take one. That can be fixed in the density analysis class.

Reply:
The thing is here we are not utilizing `FrameIteratorSliced` (which does the rewind after iteration) because we want to get accurate timing by ` self._ts = self._trajectory[i]`. So some discrepancy between AnalysisBase and ParallelAnalysisBase:
```
u = mda.Universe(GRO, XTC)
serial_analysis(u.atoms).run(stop=3)
u.trajectory.ts.frame == 0
...
parallel_analysis(u.atoms).rum(stop=3)
u.trajectory.ts.frame == 3